Marlin/src/gcode/calibrate/G34_M422.cpp

```diff
@@ -329,6 +358,11 @@ void GcodeSuite::G34() {
           #endif
         }
 
+        // Decreasing accuracy was detected so move was inverted.
+        // Will match reversed Z steppers on dual steppers. Triple will need more work to map.
```

Comment:
I was just thinking about this when I scrolled down and saw this comment.

I think we would never want the reversal logic when using known stepper positions with triple-Z.
I'll include that in the PR I send you for known stepper position fixes.